src/services/helper/sessions.js

```diff
@@ -550,6 +568,10 @@ module.exports = class SessionsHelper {
 					responseCode: 'CLIENT_ERROR',
 				})
 			}
+
```

Comment:
@ankitpws check with frontent team, they require mentor name when they do enroll and unenroll the session, If require then only add it.

Reply:
@ankitpws sending a email we may require.

Reply:
for Email only we require mentor name @rakeshSgr